docs/documentation/development/rules/ModuleTypes_schema.json

```diff
@@ -1,965 +1,1029 @@
 {
-	"$schema":"http://json-schema.org/draft-04/schema#",
```

Comment:
I was just wondering - did we agree on tabs vs spaces in JSON files already - in general I would tend to say we should do the same as for XML (as the files are packaged within the bundles), i.e. we should go for tabs. But I see that the current formatter uses spaces and thus has reformatted all your files.